tests/integration/test_main.py

```diff
@@ -47,3 +47,44 @@ def test_search_issues_no_parameters():
     assert any(
         "The 'dataset' query parameter is required" in detail for detail in details
     )
+
+
+def test_provision_summary(s3_bucket):
+    # Prepare test params
+    params = {
+        "organisation": "local-authority:BDG",
+        "offset": 0,
+        "limit": 8,
+    }
+
+    response = client.get("/performance/provision_summary", params=params)
+
+    # Validate the results from the search
+    assert response.status_code == 200
+
+    response_data = response.json()
+    assert "X-Pagination-Total-Results" in response.headers
+    assert response.headers["X-Pagination-Total-Results"] == str(17)
+    assert response.headers["X-Pagination-Limit"] == "8"
+
+    assert len(response_data) > 0
```

Comment:
It might be good to assert the first row to ensure it comes back as expected. Same for the specifications